### PR TITLE
Rename ETL master => main

### PR DESCRIPTION
**Pull request by MuriloDalRi**
https://trello.com/c/Ts2kFTYc/216-rename-etl-master-task-to-etl-main
https://trello.com/c/1DGoJ3oZ/630-rename-etl-master-task-to-etl-main-task

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: lib/tasks/etl.rake [hunk 1]
```diff
@@ -1,8 +1,8 @@
 namespace :etl do
-  desc "Run ETL master process for yesterday"
-  task master: :environment do
```
Comment (thomasleese) [inline lib/tasks/etl.rake:3]: I think we should keep the old `master` task around until the Puppet PR has been deployed so we don't end up with a period of time where the Jenkins job doesn't work. I think we can do something like this `task :master => :main`.

Comment (MuriloDalRi) [reply lib/tasks/etl.rake:3]: I've made this change. Does it look right to you?

Comment (thomasleese) [reply lib/tasks/etl.rake:3]: I think it'll need to be two tasks, so something like this:

```rb
desc "Run ETL main process for yesterday"
task main: :environment do
  unless Etl::Master::MasterProcessor.process
    abort("Etl::Master::MasterProcessor failed")
  end
end

task master: :main  # master depends on main, but does nothing itself
```

Comment (MuriloDalRi) [reply lib/tasks/etl.rake:3]: Oh right, yeah! I'll change that

**Review by thomasleese — APPROVED** [review]
Code looks fine, looks like there are some unrelated security fixes... if you want to have a go at fixing those that would be great, or we've got https://trello.com/c/6OOK52R2/2149-fix-content-data-api-brakeman-errors.

**Comment by MuriloDalRi** [comment]
> Code looks fine, looks like there are some unrelated security fixes... if you want to have a go at fixing those that would be great, or we've got https://trello.com/c/6OOK52R2/2149-fix-content-data-api-brakeman-errors.

Thanks, I was going to add a card about that! I think it should be looked at separately as it's not blocking deployment for this PR though.

**Pull request merged**